software/firmware/contrib/scope.py

```diff
@@ -0,0 +1,116 @@
+from time import sleep
+
+try:
+    # local dev
+    from software.firmware.europi import ain, b1, b2, cv1, cv2, cv4, din, k1, k2, oled
+    from software.firmware.europi import reset_state, MAX_INPUT_VOLTAGE, OLED_HEIGHT, OLED_WIDTH
```

Comment:
remove unused function

```suggestion
    from software.firmware.europi import MAX_INPUT_VOLTAGE, OLED_HEIGHT, OLED_WIDTH
```